setup.py

```diff
@@ -214,10 +214,39 @@ def finalize_options(self):
 setup(
     name="shapely",
     version=version,
+    description="Manipulation and analysis of geometric objects",
+    # description="GEOS wrapped in numpy ufuncs",
+    # description="Geometric objects, predicates, and operations",
+    long_description=descr,
+    license="BSD 3-Clause",
+    url="https://github.com/shapely/shapely",
+    keywords="geometry topology gis",
+    author="Sean Gillies, Casper van der Wel, and the Shapely contributors",
+    author_email="sean.gillies@gmail.com, caspervdw@gmail.com",
```

Comment:
It's not very clear what the requirements are around "author" (and you also have "maintainer" as alternative/additional field), for example, should it be a single person, or can it be multiple as I did here?

Some other packages put the developer mailing list as "email", but that's not something we have.

Reply:
I think it is best to sidestep this issue by not including this field at all. I strongly prefer people raising issues at github instead of emailing me.

Concerning the “author” field; I am a bit torn on this point. I like to have some credits for designing and building pygeos, but others have invested considerable amount of effort in the past years as well (like yourself @jorisvandenbossche). What warrants being listed as an “author”? 

In practice this field gets set by the creator of the project. There is something sensible in this because it is mostly the one who thought up the idea of the project in the first place. Numpy for instance has Travis Oliphaunt as author but “NumPy maintainers” as maintainer. And maintainer supersedes author in the PKG_INFO (see docs https://setuptools.pypa.io/en/latest/references/keywords.html) Related to this is https://github.com/shapely/shapely/issues/1379

Reply:
So then keeping "Sean Gillies" as `author` and add "Shapely contributors" as `maintainer`? That would be fine with me as well. 
(numpy also has " et al." after the author name)